lib/manageiq/api/client/resource.rb

```diff
@@ -25,6 +25,14 @@ def self.subclass(name)
                 @data = resource_hash.except("actions")
                 fetch_actions(resource_hash)
               end
+
+              define_method("method_missing") do |sym|
+                if data.key?(sym.to_s)
+                  data[sym.to_s]
+                else
+                  raise NoMethodError, "undefined method `#{sym}' for #{self}"
```

Comment:
Prefer `super`.  Also if you define `method_missing`, you must define `respond_to_missing?`